pkg/build/api/types.go

```diff
@@ -911,6 +915,9 @@ type GenericWebHookEvent struct {
 
 	// Env contains additional environment variables you want to pass into a builder container
 	Env []kapi.EnvVar
+
+	// StrategyOptions contains additional strategy-specific options for the build
+	StrategyOptions *BuildStrategyOptions
```

Comment:
Then `DockerStrategy *DockerStrategyOptions`.  We don't use `Options` in any field names that I see and I'm not sure it adds anything.

Let's avoid nesting.